common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -676,6 +656,11 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String NEGOTIATED_BP_VERSION_KEY = "common.broker.protocol.version.name";
 
+        /**
+         * The Boolean to send when FOCI apps are allowed to construct accounts from PRT id token in getAccounts.
+         */
+        public static final String FOCI_APPS_CAN_CONSTRUCT_ACCOUNTS_FROM_PRT_ID_TOKEN_KEY = "can.construct.accounts.from.prt.id.token";
```

Comment:
This seems like a statement rather than a question...

FOCI_APPS_CAN_CONSTRUCT_ACCOUNTS_FROM_PRT_ID_TOKEN_KEY means foci apps **CAN** construct accounts from Prt ID token

I think what you want is the following:

CAN_FOCI_APPS_CONSTRUCT_ACCOUNTS_FROM_PRT_ID_TOKEN_KEY - this would be a true boolean question